indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -172,7 +172,15 @@ public TaskStatus call()
                               command.add("-cp");
                               command.add(taskClasspath);
 
-                              Iterables.addAll(command, whiteSpaceSplitter.split(config.getJavaOpts()));
+                              Iterables.addAll(
```

Comment:
Is there a reason to explicitly remove the config values?

It seems to me having  general config values then adding in the context values last as overrides would be a reasonable behavior.